content/pages/disability-benefits/apply-for-benefits/application.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Apply for disability benefits
```

Comment:
Eventually, we will do both, but the current plan is to focus on tracking claim status for Nov. 11 because the backing APIs for filing a claim aren't ready yet.